### DISCUSSION TITLE
mpv 源代码研究

### DISCUSSION THREAD

**Discussion by lcksk** [hooke007/mpv_PlayKit#135] [mpv 通用]
是否有人研究过MPV core的源代码？想研究一下新的filter框架代码数据怎么传的，音视频同步怎么搞的 ？

**Comment by snylonue** [comment]
https://github.com/mpv-player/mpv/blob/master/DOCS/tech-overview.txt
有提到一点，上次更新是两年前了

**Reply by lcksk** [reply]
> https://github.com/mpv-player/mpv/blob/master/DOCS/tech-overview.txt 有提到一点，上次更新是两年前了
谢谢提示，这部分已经看过了。
就有点纳闷，全网那么多研究ffplay的 ，好像mpv 就没多少人有兴趣去深入研究的样子 -(
